src/main/resources/conversation/conversation.txt

```diff
@@ -23,6 +23,7 @@ what	favourite	song:I like music, my favourite song is Boogie Wonderlaaaaaaand!
 what	favourite	program	language:Java, which flows into my veins :-).	Certainly not Assembly.
 what	favourite	film:A Space Odyssey, very cool :-).
 what	favourite	animal:Cybear.. Have you got it? Cy.. bear!
+what  talk  about:I do not know, do you have any idea?	Did you know I have different games inside me?
```

Comment:
Remove this. It does not make sense for the web app

Reply:
@DS98 can you do this?

Reply:
Done